### PR TITLE
Add help paginator & other QOL updates

### PR DESCRIPTION
**Pull request by zachariapopcorn**
Due to Github being bugged for me at the moment, I can't PR multiple branches, so I gotta do this all in one PR :/

If you wish to not include any proposed feature, comment so I can revert it

Look at the commit history for updates

### CODE REVIEW

**Comment by zachariapopcorn** [comment]
Yay, there's no conflicts

**Comment by Oji0721** [comment]
Is this working?

**Comment by yogurtsyum** [comment]
@Oji0721 Generally, you should assume a pull request isn't stable yet if it isn't approved or merged.

**Comment by zachariapopcorn** [comment]
@yogurtsyum I'm finally done with this PR, may you review it?

**Comment by yogurtsyum** [comment]
It looks like you've also pushed your group ban feature.

**Comment by zachariapopcorn** [comment]
> 

No? I just added the group ban status to the ``/user`` API endpoint

File: src/commands/information/info.ts [hunk 1]
```diff
@@ -61,13 +61,7 @@ class InfoCommand extends Command {
...
         }
 
-        let robloxMember: GroupMember;
-        try {
-            robloxMember = await robloxGroup.getMember(robloxUser.id);
-            if(!robloxMember) throw new Error();
-        } catch (err) {
-            return ctx.reply({ embeds: [ getRobloxUserIsNotMemberEmbed() ]});
-        }
+        let robloxMember: GroupMember = await robloxGroup.getMember(robloxUser.id);
```
Comment (yogurtsyum) [inline src/commands/information/info.ts:64]: Why did you change this? The previous system had necessary error handling.

File: src/commands/information/help.ts [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import { discordClient } from '../../main';
```
Comment (yogurtsyum) [inline src/commands/information/help.ts:1]: Note: I've not reviewed this file yet.

File: src/config.ts [hunk 3]
```diff
@@ -60,4 +60,5 @@ export const config: BotConfig = {
         type: 'WATCHING',
         value: 'for commands.',
     },
+    pageStyleHelpCommand: true
```
Comment (yogurtsyum) [inline src/config.ts:63]: I don't know if this is necessary but it's fine for now.

**Review by yogurtsyum — COMMENTED** [review]
Looks good for now, but I've added a few comments. I will review this more later.

File: src/commands/information/info.ts (hunk 1 above)
Comment (zachariapopcorn) [reply src/commands/information/info.ts:64]: Due to group banning kicking the user, and it not allowing the checking of someone that's not in the group made doing this impossible without removing the member check, but don't worry, if a user isn't in a group, I just state that they're a Guest at rank 0 instead of letting it crash the bot

File: src/config.ts (hunk 3 above)
Comment (zachariapopcorn) [reply src/config.ts:63]: It was the cleanest way that I saw fit for doing this implementation

Comment (yogurtsyum) [reply src/config.ts:63]: I meant I don't know if it needs to be a toggle.

Comment (zachariapopcorn) [reply src/config.ts:63]: I wanted to keep the new and old functionality, as some people prefer a pagination based help menu while others prefer the long embed version

Comment (zachariapopcorn) [reply src/config.ts:63]: And if not a config toggle, what would it be then?

**Comment by zachariapopcorn** [comment]
So what'll happen with this PR?

**Comment by yogurtsyum** [comment]
I'll just remove the 1 feature that was implemented already. Sorry I couldn't use this one, but I was reviewing this and noticed some issues, so I figured I would just push my own solution for now because I needed it done.

And for the review for this, I will have it posted sometime tomorrow.

**Comment by zachariapopcorn** [comment]
Alright

**Comment by zachariapopcorn** [comment]
I wonder, should I just remake the PR excluding the features in other PRs/in the bot already?
